src/components/Auth/Login/Login.react.js

```diff
@@ -15,6 +19,28 @@ import { slide as Menu } from 'react-burger-menu';
 /* eslint-disable */
 const cookies = new Cookies();
 
+const ListMenu = () => (
+					<IconMenu className='IconMenu'
```

Comment:
Why so many empty lines. Is there a specific reason. If not please consider removing them. thanks